### DISCUSSION TITLE
Map - text alignment is not working

### DISCUSSION THREAD

**Discussion by SAKTHIVELMARIMUTHU** [GrapesJS/grapesjs#5727] [Q&A] [answered]
As i said the allignment for the map is not working

**Comment by mosh-tudor** [comment]
@SAKTHIVELMARIMUTHU Which text? What do you mean?

**Comment by SAKTHIVELMARIMUTHU** [comment]
@mosh-tudor if a map is dragged inside the cell or a row I cannot be able to align the map like we can align the images and heading and text

**Reply by mosh-tudor** [reply, marked as answer]
@SAKTHIVELMARIMUTHU 
The map component is not text to be aligned with "text-align" style. 

Block elements are usually aligned horizontally in two ways:
1. `margin-right: auto; margin-left: auto;` -> OR ->
2. on parent element: `display: flex; justify-content: center;`

Just google "align div horizontally".

**Reply by SAKTHIVELMARIMUTHU** [reply]
@mosh-tudor thank you i think now I got the picture
